packages/toolkit/src/view/knowledge/KnowledgeBaseView.tsx

```diff
@@ -0,0 +1,145 @@
+import { GeneralAppPageProp } from "../../lib";
+import { Sidebar } from "./components/Sidebar";
+import { KnowledgeBaseTab } from "./components/KnowledgeBaseTab";
+import { UploadExploreTab } from "./components/UploadExploreTab";
+import { CatalogFilesTab } from "./components/CatalogFilesTab";
+import { MarkdownTab } from "./components/MarkdownTab";
+import * as React from "react";
+import { KnowledgeBase } from "../../lib/vdp-sdk/knowledge/types";
+// import { Button, Icons, LinkButton } from "@instill-ai/design-system";
+import { DELETE_KNOWLEDGE_BASE_TIMEOUT, CREDIT_TIMEOUT } from "./components/undoDeleteTime";
+import { ChunkTab } from "./components/ChunkTab";
+import { ImageTab } from "./components/ImageTab";
+import { Nullable } from "@instill-ai/toolkit";
+import { RetrieveTestTab } from "./components/RetrieveTestTab";
+import CreditUsageNotification from "./components/Notifications/CreditUsageFileNotification";
```

Comment:
@thewbuk This import page is a bit messy. 

You can do several things to it

- In component folder, have a index.ts file export all the component filed you need
- Try to export file layer by layer

The ultimate goal is to import only from "./components" alone

And we don't allow default export here, so something like import CreditUsageNotification from "..." need to be revamped with named export